argus.js

```diff
@@ -3,21 +3,33 @@ const MESSAGE_ARGUS_TOKEN_REQUEST = "argus-token-request";
 const MESSAGE_ARGUS_TOKEN_RESPONSE = "argus-token-response";
 
 const ArgusJS = function(token, mdrUrl) {
-    let currentToken = token;
+    let auth
```

Comment:
can you add error messaging 
if a token is null / undefinted or without .acesss it will fail silently. and document what the expected format to use in the readme